mrtt-ui/package.json

```diff
@@ -60,7 +72,11 @@
     "production": [
       ">0.2%",
       "not dead",
-      "not op_mini all"
+      "not op_mini all",
+      "not ie 11",
+      "not chrome < 51",
+      "not safari < 10",
+      "not android < 51"
```

Comment:
Workaround for transpilation errors/warnings: https://docs.mapbox.com/mapbox-gl-js/guides/install/#transpiling